erpnext/agriculture/doctype/plant_batch/plant_batch.js

```diff
@@ -46,8 +50,41 @@ frappe.ui.form.on('Plant Batch', {
 				});
 			}, __("Create"));
 		}
-
-	}
+	},
+	split_plant_batch: function(frm) {
+		frappe.prompt([{
+			fieldname: 'split_count',
+			label: __('New Plant Batch Untracked Count'),
+			fieldtype: 'Int',
```

Comment:
I think this should be mandatory. Check if METRC lets you create empty plant batches.